.azure/app-cloud-e2e.yml

```diff
@@ -133,6 +133,14 @@ jobs:
       condition: eq(variables['name'], 'quick_start')
       displayName: 'Install Quick Start'
 
+    - bash: python -m pip install -q -r .actions/requirements.txt
+      displayName: 'Install assistant dependencies'
+
+    - bash: |
+        python .actions/assistant.py copy_replace_imports --source_dir="./examples" --source_import="lightning.app" --target_import="lightning_app"
+        python .actions/assistant.py copy_replace_imports --source_dir="./examples" --source_import="lightning" --target_import="lightning_app"
```

Comment:
Why do we want the standalone imports here instead of the `lightning` alternatives?

Reply:
because we are testing App as standalone which means that `lightning` does not exists

Reply:
Okay. I guess this will change when we parametrize this file